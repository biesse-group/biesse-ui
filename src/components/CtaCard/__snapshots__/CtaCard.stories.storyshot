// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Cards/CtaCard Full Image 1`] = `
.c5 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #252525;
}

.c5 a {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #252525;
}

.c9 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 10px;
  width: 10px;
  -webkit-flex: 0 0 10px;
  -ms-flex: 0 0 10px;
  flex: 0 0 10px;
  color: inherit;
}

.c9 > svg {
  height: 100%;
  width: 100%;
  color: inherit;
}

.c10 {
  margin-left: 6px;
  display: inline;
}

.c8 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: bold;
  border: 0;
  border-radius: 50px;
  cursor: pointer;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  line-height: 1;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
  text-transform: uppercase;
  outline: none !important;
  white-space: nowrap;
  font-size: 14px;
  padding: 0px 26px;
  height: 30px;
  background: transparent;
  color: #6D6E70;
  text-transform: none;
  padding: 0px;
}

.c1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  cursor: auto;
  height: 100%;
  width: 100%;
  overflow: hidden;
  border-bottom-left-radius: 40px;
  border-top-right-radius: 40px;
  background-color: #F5F5F5;
  -webkit-transition: all 0.5s ease-out;
  transition: all 0.5s ease-out;
}

.c1:hover {
  background-color: #FFF;
  box-shadow: 0 0 20px 0 rgba(140,140,140,0.5);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 0 0 30%;
  -ms-flex: 0 0 30%;
  flex: 0 0 30%;
}

.c4 {
  overflow-y: hidden;
  padding: 20px 50px 20px 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  padding-bottom: 10px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c6 {
  line-height: 20px;
}

.c7 {
  padding-top: 15px;
  margin-top: auto;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c0 {
  max-width: 600px;
}

.c3 {
  max-height: 100%;
  max-width: 100%;
  padding: 10px 10px 10px 30px;
}

@media screen and (max-width:991px) {
  .c5 {
    font-size: 15px;
  }
}

@media screen and (max-width:767px) {
  .c5 {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {
  .c5 a {
    font-size: 15px;
  }
}

@media screen and (max-width:767px) {
  .c5 a {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {
  .c2 {
    -webkit-flex: 0 0 100%;
    -ms-flex: 0 0 100%;
    flex: 0 0 100%;
  }
}

@media screen and (max-width:991px) {
  .c4 {
    padding-right: 20px;
  }
}

@media screen and (max-width:991px) {
  .c4 {
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media screen and (max-width:991px) {
  .c4 {
    display: none;
  }
}

@media screen and (max-width:991px) {
  .c6 {
    display: none;
  }
}

@media screen and (max-width:991px) {
  .c0 {
    height: 96px;
    max-width: 360px;
  }
}

@media screen and (max-width:767px) {
  .c0 {
    height: 84px;
  }
}

@media screen and (max-width:991px) {
  .c3 {
    padding: 20px;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <img
        alt="kuka logo"
        className="c3"
        src="/assets/kuka-logo.png"
      />
    </div>
    <div
      className="c4"
    >
      <span
        className="c5 c6"
        color="dark"
      >
        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam eaque ipsa...
      </span>
      <div
        className="c7"
      >
        <button
          className="c8 "
          size="small"
          type="button"
        >
          Scopri
          <span
            className="c9 c10"
            size="10px"
          >
            <svg
              viewBox="0 0 50 50"
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <defs>
                <path
                  d="M0 0h25.722v48H0z"
                />
              </defs>
              <g
                fill="none"
                fillRule="evenodd"
                transform="translate(12 1)"
              >
                <path
                  d="M1.768 48c-.26 0-.521-.043-.752-.16-1.134-.578-1.323-1.964-.543-2.807l21.408-20.659L.471 2.965A1.758 1.758 0 0 1 .522.518 1.756 1.756 0 0 1 1.77 0c.446 0 .872.167 1.2.47l22.21 22.56c.729.74.72 1.93-.02 2.657L2.97 47.53c-.324.302-.752.47-1.2.47h-.002Z"
                  fill="currentColor"
                />
              </g>
            </svg>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Cards/CtaCard With Title 1`] = `
.c7 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #252525;
}

.c7 a {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #252525;
}

.c11 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 10px;
  width: 10px;
  -webkit-flex: 0 0 10px;
  -ms-flex: 0 0 10px;
  flex: 0 0 10px;
  color: inherit;
}

.c11 > svg {
  height: 100%;
  width: 100%;
  color: inherit;
}

.c12 {
  margin-left: 6px;
  display: inline;
}

.c10 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: bold;
  border: 0;
  border-radius: 50px;
  cursor: pointer;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  line-height: 1;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
  text-transform: uppercase;
  outline: none !important;
  white-space: nowrap;
  font-size: 14px;
  padding: 0px 26px;
  height: 30px;
  background: transparent;
  color: #6D6E70;
  text-transform: none;
  padding: 0px;
}

.c5 {
  margin-top: 0;
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 700;
  font-size: 34px;
  text-transform: uppercase;
  color: #6D6E70;
}

.c1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  cursor: auto;
  height: 100%;
  width: 100%;
  overflow: hidden;
  border-bottom-left-radius: 40px;
  border-top-right-radius: 40px;
  background-color: #F5F5F5;
  -webkit-transition: all 0.5s ease-out;
  transition: all 0.5s ease-out;
}

.c1:hover {
  background-color: #FFF;
  box-shadow: 0 0 20px 0 rgba(140,140,140,0.5);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 0 0 30%;
  -ms-flex: 0 0 30%;
  flex: 0 0 30%;
}

.c4 {
  overflow-y: hidden;
  padding: 20px 50px 20px 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  padding-bottom: 10px;
}

.c6 {
  margin-bottom: 14px;
}

.c8 {
  line-height: 20px;
}

.c9 {
  padding-top: 15px;
  margin-top: auto;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c0 {
  max-width: 600px;
}

.c3 {
  border-bottom-left-radius: 40px;
  border-top-right-radius: 40px;
  background-color: #6D6E70;
  height: 100%;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media screen and (max-width:991px) {
  .c7 {
    font-size: 15px;
  }
}

@media screen and (max-width:767px) {
  .c7 {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {
  .c7 a {
    font-size: 15px;
  }
}

@media screen and (max-width:767px) {
  .c7 a {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {
  .c5 {
    font-size: 30px;
  }
}

@media screen and (max-width:767px) {
  .c5 {
    font-size: 25px;
  }
}

@media screen and (max-width:991px) {
  .c2 {
    -webkit-flex: 0 0 40%;
    -ms-flex: 0 0 40%;
    flex: 0 0 40%;
  }
}

@media screen and (max-width:991px) {
  .c4 {
    padding-right: 20px;
  }
}

@media screen and (max-width:991px) {
  .c4 {
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media screen and (max-width:991px) {
  .c6 {
    font-size: 30px;
    margin-bottom: 0px;
  }
}

@media screen and (max-width:767px) {
  .c6 {
    font-size: 20px;
  }
}

@media screen and (max-width:991px) {
  .c8 {
    display: none;
  }
}

@media screen and (max-width:991px) {
  .c0 {
    height: 96px;
    max-width: 360px;
  }
}

@media screen and (max-width:767px) {
  .c0 {
    height: 84px;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <div
        className="c3"
      >
        <img
          alt="product"
          src="/assets/product-carousel-2.png"
          style={
            Object {
              "maxHeight": "100%",
              "maxWidth": "100%",
            }
          }
        />
      </div>
    </div>
    <div
      className="c4"
    >
      <h3
        className="c5 c6"
        color="primary"
        size="md"
      >
        ES951
      </h3>
      <span
        className="c7 c8"
        color="dark"
      >
        Sed ut perspiciatis unde omnis iste natus error sit voluptatem
      </span>
      <div
        className="c9"
      >
        <button
          className="c10 "
          size="small"
          type="button"
        >
          Scopri
          <span
            className="c11 c12"
            size="10px"
          >
            <svg
              viewBox="0 0 50 50"
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <defs>
                <path
                  d="M0 0h25.722v48H0z"
                />
              </defs>
              <g
                fill="none"
                fillRule="evenodd"
                transform="translate(12 1)"
              >
                <path
                  d="M1.768 48c-.26 0-.521-.043-.752-.16-1.134-.578-1.323-1.964-.543-2.807l21.408-20.659L.471 2.965A1.758 1.758 0 0 1 .522.518 1.756 1.756 0 0 1 1.77 0c.446 0 .872.167 1.2.47l22.21 22.56c.729.74.72 1.93-.02 2.657L2.97 47.53c-.324.302-.752.47-1.2.47h-.002Z"
                  fill="currentColor"
                />
              </g>
            </svg>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
