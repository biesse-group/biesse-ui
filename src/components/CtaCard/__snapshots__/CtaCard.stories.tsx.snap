// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards/CtaCard FullImage smoke-test 1`] = `
<div class="sc-fPXMVe chJXyq">
  <div class="sc-cwHptR eliQlV">
    <div class="sc-jEACwC kTSwme">
      <img src="/assets/kuka-logo.png"
           alt="kuka logo"
           class="sc-ikkxIA hTUOUH"
      >
    </div>
    <div class="sc-cPiKLX ismBqR">
      <span color="dark"
            class="sc-dcJsrY hQjSgU sc-eDPEul gQZBUs"
      >
        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam eaque ipsa...
      </span>
      <div class="sc-eldPxv lkqoeG">
        <button type="button"
                class="sc-fqkvVR gCepkH sc-jlZhew"
        >
          Scopri
          <span class="sc-aXZVg hqNmJX sc-eqUAAy kVWemk">
            <svg viewbox="0 0 50 50"
                 xmlns="http://www.w3.org/2000/svg"
                 xmlns:xlink="http://www.w3.org/1999/xlink"
            >
              <defs>
                <path d="M0 0h25.722v48H0z">
                </path>
              </defs>
              <g transform="translate(12 1)"
                 fill="none"
                 fill-rule="evenodd"
              >
                <path d="M1.768 48c-.26 0-.521-.043-.752-.16-1.134-.578-1.323-1.964-.543-2.807l21.408-20.659L.471 2.965A1.758 1.758 0 0 1 .522.518 1.756 1.756 0 0 1 1.77 0c.446 0 .872.167 1.2.47l22.21 22.56c.729.74.72 1.93-.02 2.657L2.97 47.53c-.324.302-.752.47-1.2.47h-.002Z"
                      fill="currentColor"
                >
                </path>
              </g>
            </svg>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Cards/CtaCard WithTitle smoke-test 1`] = `
<div class="sc-fPXMVe chJXyq">
  <div class="sc-cwHptR eliQlV">
    <div class="sc-jEACwC YfAHn">
      <div class="sc-gFqAkR cQLLy">
        <img src="/assets/product-carousel-2.png"
             alt="product"
             style="max-width: 100%; max-height: 100%;"
        >
      </div>
    </div>
    <div class="sc-cPiKLX ftFjwQ">
      <h3 color="primary"
          class="sc-imWYAI dCVdfl sc-dLMFU huvzey"
      >
        ES951
      </h3>
      <span color="dark"
            class="sc-dcJsrY hQjSgU sc-eDPEul gQZBUs"
      >
        Sed ut perspiciatis unde omnis iste natus error sit voluptatem
      </span>
      <div class="sc-eldPxv lkqoeG">
        <button type="button"
                class="sc-fqkvVR gCepkH sc-jlZhew"
        >
          Scopri
          <span class="sc-aXZVg hqNmJX sc-eqUAAy kVWemk">
            <svg viewbox="0 0 50 50"
                 xmlns="http://www.w3.org/2000/svg"
                 xmlns:xlink="http://www.w3.org/1999/xlink"
            >
              <defs>
                <path d="M0 0h25.722v48H0z">
                </path>
              </defs>
              <g transform="translate(12 1)"
                 fill="none"
                 fill-rule="evenodd"
              >
                <path d="M1.768 48c-.26 0-.521-.043-.752-.16-1.134-.578-1.323-1.964-.543-2.807l21.408-20.659L.471 2.965A1.758 1.758 0 0 1 .522.518 1.756 1.756 0 0 1 1.77 0c.446 0 .872.167 1.2.47l22.21 22.56c.729.74.72 1.93-.02 2.657L2.97 47.53c-.324.302-.752.47-1.2.47h-.002Z"
                      fill="currentColor"
                >
                </path>
              </g>
            </svg>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
